index.bs

```diff
@@ -1114,6 +1130,30 @@ The <dfn>context type</dfn> is the type of the execution context that manages th
 <dd>Context created from WebGPU device.</dd>
 </dl>
 
+<div algorithm>
+The <dfn attribute for=MLContext>accelerated</dfn> getter steps are to return [=this=].{{MLContext/[[accelerated]]}}.
+</div>
+
+<div algorithm>
+    The <dfn attribute for=MLContext>cpuFallbackActive</dfn> getter steps are:
+        1. If [=this=].{{MLContext/[[cpuFallbackActive]]}} is `undefined`, then invoke [=poll CPU fallback status=].
+        1. Return [=this=].{{MLContext/[[cpuFallbackActive]]}}.
+</div>
+
+<details open algorithm>
+  <summary>
+    To <dfn>poll CPU fallback status</dfn>, run the following steps.
+  </summary>
+    1. If [=this=].{{MLContext/[[accelerated]]}} is `false`, then:
+        1. Set [=this=].{{MLContext/[[cpuFallbackActive]]}} to `true` and return.
+    1. Issue a request to check whether the underlying platform uses CPU as the main underlying execution device for inference. If yes, then implementations <span class=allow-2119>should</span> set [=this=].{{MLContext/[[cpuFallbackActive]]}} to `true` and return.
```

Comment:
Is "issue a request" achievable in a synchronous manner? If not perhaps this should be an async method.